inframanager/api_handler/api_handler.go

```diff
@@ -159,48 +161,64 @@ func CloseGNMIConn() {
 	}
 }
 
-func getPortID(ifName string) (portID uint32, err error) {
-	var resp *pb.GetResponse
+func getPortID(ifName string, macAddr net.HardwareAddr) (portID uint32, err error) {
```

Comment:
This will need to go in DPDK and MEV tagged files. As this part is done using gnmi in DPDK and second octet in MEV

Reply:
Since the function gets the ID based upon interface type, we will uncomment below code to allow it to work for both TAP and CDQ interface types. Additionally, the interface "types" argument needs to be passed.

Reply:
Leaving this comment for Venkatest to address as separate commit on this branch.